meetings/2018-06-13.md

```diff
@@ -0,0 +1,110 @@
+  * Nothing to add since last board meeting
+  * Work on getting CommComm charter in place for Director etc.
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * Modules
+    * Have gotten through a larger number of the use cases
+    * PR open for meta require
+    * Starting to talk about deadlines
+    * Main contentious topics is about transparent interop, module specifier resolution
+    * Package name maps proposal from Domenic, to allow install time tooling to generate
+      manifest to be consume by browser to allow bare imports.  May provide path to being
+      to use modules in both browsers and Node.js without changes. Does not affect CJS
+
+  * N-API
+     * Continued push on doc for node-addon-api
+     * Looks like we’ll have workshop and a few talks at Node Summit
+     * Proposal for adding new APIs as experimental
+     * Proposal for guidance on adding new APIs
+
+  * OpenSSL Evolution
+    * Pull request 21282, would some some additional opinion.  Security issue with CVE
+      Details are public but not in release as its low severity.
+    * Anna, would this be considered a breaking change if not a security issue?
+
+  * Workers
+    * Not too much new since PR landed, may include in next 10.x release (it is behind a flag)
+    * incorning out minor issues.
+
+  * TSC Governance
+    * Some discussion on code owners, mixed feelings
+    * Discussion around backports, and our definition of done.  Should we not close
+      pull requests until all backports audited, landed etc.
+
+  * New Streams APIs
+    * Progress at collab summit, Jerimiah, fs-source and fs-sync modules, more status
+      In fishrock123/bob repo.   Nothing new since collab summit.
+
+* Proposal: add all new core modules under a scope? (too late for http2) [#389](https://github.com/nodejs/TSC/issues/389)
+  * Myles, we need namespaces and we need to move on faster than we have to. Every
+    time we add something new have have the shadowing issue with npm
+  * We have a number of PRs open (mime, brotili) and other recent ones have had
+    collisions.  Also was issue for fs.promises.
+  * Questions about what kind of symbol we use, Myles highly recommends @nodejs
+    namespace
+  * TC39 is also looking into namespaces as a language feature, advice from committee was
+    not to wait for it (there are no proposals now, likely more than a year out).
+  * Can’t just be a directory, security concern about modules that don’t have the namespace
+    may be an issue.
+  * Jeremiah only move to namespaces once.  Also align with returning promisified APIs.
+  * Myles that could be a 11/2 year to 2 years from now.
+  * Let’s leave on the agenda.
+  * Anna, may make sense to ask if we have enough engagement from those outside the our
+    bubble.
+    * single sentence question -> should node namespace modules? along with context.
```

Comment:
So what are the action items here? Last time it was “wait a week and revisit”

Reply:
I think the next action is to continue the discussion in the issue tracker.  There is no clear consensus in the TSC members one way or the other. In addition, I don't think there was the consensus that we needed to rush and have a vote either as the feeling is that there is still work to be done as part of our consensus seeking model.  In particular, I think I remember a request to better frame the pros/cons and move people toward a shared understanding  (for example it being harder to add modules is seen but a con by some and a pro by others).  If anybody else had some details on next actions to add please comment.

Reply:
Just to add,  the week before not enough people had reviewed the issue to have a discussion, this time there was a good discussion, but not one that resulted in a decision one way or the other or agreement it had reached the point were a vote was appropriate.

Reply:
What I'm hoping to have is a TSC discussion that some kind of "non-bare" identifier is needed for all new core modules.

The details of that certainly can be hashed out in GitHub.